scripts/validate.sh

```diff
@@ -48,10 +48,16 @@ fi
 echo "Verifying that the package versionlocks are correct..."
 
 function versionlock-entries() {
-  # the format of this output is EPOCH:NAME-VERSION-RELEASE.ARCH
-  # more info in yum-versionlock(1)
-  # rpm doesn't accept EPOCH when querying the db, so remove it
-  yum versionlock list --quiet | cut -d ':' -f2
+  if [ -f "/etc/dnf/plugins/versionlock.conf" ]; then
+    # the format of this output in al2023 is NAME-EPOCH:VERSION-RELEASE.ARCH
+    # rpm works fine with this format
+    yum versionlock list --quiet
```

Comment:
this reminds me -- I don't think we need to do this versionlocking at all on AL2023. The repo itself is kind of "version locked" in that distro, and the recommendation I've heard from AL folks was to get rid of this. It's mostly been more annoying than helpful, too.

Reply:
Correct. By default, AL2023 images are locked to a specific snapshot of the repo.

The only reason to use a dnf plugin (note that `yum` is not a thing on AL2023, only a symlink exists for compatibility) for locking versions is when you want to move everything *else* forward but to lock some other things... which should all be a VERY exceptional scenario.

Reply:
okeydokey, adding an if condition to avoid running the versionlocking code